src/main/webapp/js/pages/index.js

```diff
@@ -34,10 +34,41 @@ document.querySelector('.course-list').addEventListener('click', () => {
       course => {sections2[course] = courseInfo[course].sections[1]});
   const schedules = [sections1, sections2];
 
-  // TODO(ramyabuva): add pagination for multiple schedules
-  selected.forEach(course => {
-    Calendar.addCourse(courseInfo[course], schedules[0][course]);
-  });
+  function addScheduleToCalendar(schedule){
```

Comment:
instead of nesting this function declaration, can this go outside as a helper function?

Reply:
As it is, it can't go outside of the function because it uses variables inside the scope of the function. I could add those variables as parameters instead and move the nested function outside?

Reply:
Yep, that sounds reasonable.